hluti2.sql

```diff
@@ -0,0 +1,58 @@
+CREATE OR REPLACE VIEW lannister.v_pov_characters_human_readable AS
+WITH characters AS (
+    SELECT
+        CONCAT(c.titles[1], ' ', c.name) AS full_name,
```

Comment:
Hér bætist við óþarfa bil ef það er ekki neinn titill til staðar

```suggestion
concat_ws(' '::text, c.titles[1], c.name)
```